draft-ietf-quic-transport.md

```diff
@@ -2697,6 +2697,43 @@ also be forward-secure encrypted.  Since the attacker will not have the forward
 secure key, the attacker will not be able to generate forward-secure encrypted
 packets with ACK frames.
 
+## Stream fragmentation and reassembly attacks
+
+An adversarial client may attempt to
```

Comment:
This is any endpoint, though I agree that it's (usually) not very interesting for a server to mount the attack.